apps/frontend/src/components/global/upload_tabs/aws/AuthStepBasic.vue

```diff
@@ -11,11 +11,9 @@
       <v-text-field
         :value="secret_token"
         label="User Account Secret Token"
+        type="password"
         :rules="[req_rule]"
-        :append-icon="show_secret ? 'mdi-eye' : 'mdi-eye-off'"
-        :type="show_secret ? 'text' : 'password'"
         @input="change_secret_token"
-        @click:append="show_secret = !show_secret"
```

Comment:
Since `show_secret` is no longer used, the data field can be removed from this Component.